CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 3.21
+
+* Highlight clean code attributes and impacts on software qualities in rule descriptions
```

Comment:
I wonder if we should  add a documentation link here to help the user understand what we mean by CCA/impact/sq
(It wouldn't be the first time we add a documentation link in changelog )

Reply:
@marco-comi-sonarsource do you think it would make sense to add a link to the clean code concepts documentation in the changelog?

Reply:
Yes, being those new concepts that could help users learn about them.